core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -195,6 +198,7 @@ angular.module('oppia').directive('topNavigationBar', [
           };
 
           ctrl.$onInit = function() {
+            $scope.SHOW_CLASSROOM_CALLOUT = SHOW_CLASSROOM_CALLOUT;
```

Comment:
Seems we can do this much earlier, right? Let's move it to line 51, please.

Reply:
Yes, sure! I'll change it update asap.